cli/src/main.rs

```diff
@@ -905,6 +905,7 @@ fn optimize_and_output<T: FieldElement>(file: &str) {
 mod test {
     use crate::{run_command, Commands, CsvRenderModeCLI, FieldArgument};
     use powdr_backend::BackendType;
+    use test_log::test;
```

Comment:
why?

Reply:
Because there was a dependency but it wasn't being used. This allows for `RUST_LOG` to work on tests.